core/src/i18n/en.yaml

```
@@ -514,12 +514,23 @@ dnsentries:
     ttl: TTL
     dns-name: DNS Name
     text: Text
+    target: Target
     targets: Targets
+    use-cname: Use CNAME
+  name_singular: DNS Entry
   placeholders:
-    targets: Target records (CNAME or A records), one per line
-    text: Text records, one per line
     ttl: Enter the time to live.
-    dns-name: Enter DNS Name
+    dns-name: Enter the DNS Name.
+    target-a: Enter the A record Target.
+    target-cname: Enter the CNAME record Target.
+  tooltips:
+    use-cname: Switch to CNAME record.
+    use-a: Switch to A record.
+    target: Target must be an A record, or a CNAME record.
+    targets: Targets records, either target or text are required.
```

```suggestion
    targets: Targets records, either target or text, are required.
```

Is it really "target**s** records", or rather "target records" ?